### DISCUSSION TITLE
The DNS record of the return path remains default and is not changing to the entry configured in the postal.yaml file.

### DISCUSSION THREAD

**Discussion by faheemkhan5** [postalserver/postal#2981] [Help with using Postal]
Please Help, 

As mentioned in the subject. i have installed the fresh Postal mail server. The DNS record of the return path remains default and is not changing to the entry configured in the postal.yaml file.  

![image](https://github.com/postalserver/postal/assets/40638668/bedaa73f-16c6-4aa2-8ae0-87143831c923)

**Comment by willpower232** [comment]
If you have updated the config file, you need to restart Postal.

If you haven't updated the config file, you need to update the config file (and then restart Postal)

**Reply by faheemkhan5** [reply]
The config file has been updated and the Postal server has been restarted several times, but the same issue persists. I even deleted all the Docker images and reinstalled them with the correct config file.

**Reply by willpower232** [reply]
Hopefully a stupid question but you have removed every reference to example.com from the config file and replaced with your own subdomains?

If so, it would be worth doing something like `postal bash web` and `cat /config/postal.yml` to confirm that your updated config file is making it inside the docker container.

**Reply by vikasmic** [reply]
Check the syntax of the parameter. with new version, we have updated syntax of key value pair
